include/openenclave/bits/sgx/sgxtypes.h

```diff
@@ -47,6 +47,8 @@ OE_EXTERNC_BEGIN
 #define SGX_SECINFO_TCS 0x0000000000000000100
 #define SGX_SECINFO_REG 0x0000000000000000200
 
+#define SGX_SECS_MISCSELECT_EXINFO 0x0000000000000001UL
```

Comment:
This should be moved to near line 134 where `SGX_SIGSTRUCT_MISCMASK` is defined.

Reply:
See my comments above.